MACOSX.md

```diff
@@ -22,14 +22,82 @@ Homebrew 1.6.0
 Homebrew/homebrew-core (git revision 66e9; last commit 2018-04-11)
 ```
 
+Update using:
+
+```
+$ brew update
+```
+
+Fix formulas for `openldap` and `apr-util`
+
+```
+$ cd /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula
+$ # edit apr-util.rb and openldap.rb to make the below diffs
+$ git diff
+diff --git a/Formula/apr-util.rb b/Formula/apr-util.rb
+index 4dee25282..97f460398 100644
+--- a/Formula/apr-util.rb
++++ b/Formula/apr-util.rb
+@@ -5,24 +5,28 @@ class AprUtil < Formula
+   sha256 "d3e12f7b6ad12687572a3a39475545a072608f4ba03a6ce8a3778f607dd0035b"
+   revision 1
+ 
+-  bottle do
+-    sha256 "e4927892e16a3c9cf0d037c1777a6e5728fef2f5abfbc0af3d0d444e9d6a1d2b" => :mojave
+-    sha256 "1bdf0cda4f0015318994a162971505f9807cb0589a4b0cbc7828531e19b6f739" => :high_sierra
+-    sha256 "75c244c3a34abab343f0db7652aeb2c2ba472e7ad91f13af5524d17bba3001f2" => :sierra
+-    sha256 "bae285ada445a2b5cc8b43cb8c61a75e177056c6176d0622f6f87b1b17a8502f" => :el_capitan
+-  end
+ 
```

Comment:
I think we need to explain why the bottle paragraph is removed.
Also do these instructions only work for Mojave?

Reply:
The instructions should work for Sierra, High Sierra and Mojave.
Bottle is removed so that Homebrew does not download a prebuilt binary.
I'll update to explain why the patching has to happen which will include the why of removing the bottles.

Reply:
Regarding the log location. How did you install httpd on your Mac?

Reply:
I've updated the text.